src/Simplex/Chat/Messages.hs

```diff
@@ -544,6 +548,8 @@ data RcvGroupEvent
   = RGEMemberAdded {groupMemberId :: GroupMemberId, profile :: Profile} -- CRJoinedGroupMemberConnecting
   | RGEMemberConnected -- CRUserJoinedGroup, CRJoinedGroupMember, CRConnectedToGroupMember
   | RGEMemberLeft -- CRLeftMember
+  | RGEMemberRole {groupMemberId :: GroupMemberId, role :: GroupMemberRole}
```

Comment:
Above comments would probably require profile in this and SGEMemberRole. I think it'd be necessary in UIs as well